run.sh

```diff
@@ -1,15 +1,58 @@
 #!/bin/bash
 set -e
 set -x
-export DEVPI_SERVERDIR=/mnt
-export DEVPI_CLIENTDIR=/tmp/devpi-client
-[[ -f $DEVPI_SERVERDIR/.serverversion ]] || initialize=yes
-
-devpi-server --start --host 0.0.0.0 --port 3141
-if [[ $initialize = yes ]]; then
-  devpi use http://localhost:3141
-  devpi login root --password=''
-  devpi user -m root password="${DEVPI_PASSWORD}"
-  devpi index -y -c public pypi_whitelist='*'
-fi
-tail -f $DEVPI_SERVERDIR/.xproc/devpi-server/xprocess.log
+
+# Default environment if none is provided
+: ${DEVPI_SERVERIP:=127.0.0.1}
+: ${DEVPI_SERVERPORT:=3143}
+: ${DEVPI_SERVERDIR:=/mnt}
+: ${DEVPI_CLIENTDIR:=/tmp/devpi-client}
+: ${DEVPI_CLIENTPASSWORD:=""}
```

Comment:
Remember the image should be backwards compatible for previously used name `DEVPI_PASSWORD` (still mentioned in README)

just replace this line by `: ${DEVPI_CLIENTPASSWORD:=$DEVPI_PASSWORD}`